app/code/Magento/Customer/Test/Mftf/ActionGroup/StorefrontGoToCustomerAddressesPageActionGroup.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="StorefrontGoToCustomerAddressesPageActionGroup">
+        <annotations>
+            <description>Goes to the storefront customer addresses page</description>
+        </annotations>
+        <amOnPage url="{{StorefrontCustomerAddressesPage.url}}" stepKey="goToAddressPage"/>
+        <waitForPageLoad stepKey="waitForPageLoad"/>
```

Comment:
`<amOnPage` already includes `waitForPageLoad` in its implementation, you don't need to add it manually.

Reply:
done